### PR TITLE
Cart ddb

### PR DESCRIPTION
**Pull request by ashleyqw**
Added cart folder with api and builder subfolders. The cart.js and serverless.yml files read cart events from the Kinesis stream and populate a new cart DynamoDB.

### CODE REVIEW

File: cart/builder/cart.js [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+  // resources
+  TABLE_CART_NAME: process.env.TABLE_CART_NAME,
+}
+
+const kh = new KH.KinesisHandler(eventSchema, constants.MODULE)
+
+const dynamo = new aws.DynamoDB.DocumentClient()
+
+const impl = {
+  // TODO update the below comment to reflect cart schema
```
Comment (sainap) [inline cart/builder/cart.js]: Can you update the schema so it matches a carted item event?

File: cart/builder/cart.js [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+        '#cb=if_not_exists(#cb,:cb),',
+        '#id=:id',
+      ].join(' '),
+      ExpressionAttributeNames: {
+        '#c': 'created',
+        '#cb': 'createdBy',
+        '#id': 'productId',
+      },
+      ExpressionAttributeValues: {
+        ':c': updated,
```
Comment (sainap) [inline cart/builder/cart.js]: you should just be able to say Date.now() here, no need to create a new parameter, unless you are planning to use it later :)

File: cart/builder/cart.js [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+  // resources
+  TABLE_CART_NAME: process.env.TABLE_CART_NAME,
+}
+
+const kh = new KH.KinesisHandler(eventSchema, constants.MODULE)
+
+const dynamo = new aws.DynamoDB.DocumentClient()
+
+const impl = {
+  // TODO update the below comment to reflect cart schema
```
Comment (glee2244) [inline cart/builder/cart.js]: Might need to update this?

Comment (glee2244) [reply cart/builder/cart.js]: Will update this comment in the next PR

**Pull request merged**